lib/client.js

```diff
@@ -45,9 +45,9 @@ var Client = function (config) {
 		return response;
 	}
 
-	function getUserAgentHeader() {
-		return packageInfo.name + "-v" + packageInfo.version;
-	}
+	this.makeRequest = function (params) {
```

Comment:
Yeah I didn't even actually make those changes. It was automatically done when I used the command to fix styling.